app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/CommonTriageQuestionsController.scala

```diff
@@ -396,6 +398,25 @@ class CommonTriageQuestionsController @Inject() (
       }
     }
 
+  private def amendReturnDisposalDateBackLink(state: Either[StartingNewDraftReturn, FillingOutReturn]): Call =
+    state.fold(
+      _ => controllers.routes.StartController.start(),
+      _.draftReturn.fold(
+        _ => routes.MultipleDisposalsTriageController.whenWereContractsExchanged(),
+        _ => routes.SingleDisposalsTriageController.whenWasDisposalDate(),
+        _ => routes.SingleDisposalsTriageController.disposalDateOfShares(),
+        _ => routes.MultipleDisposalsTriageController.disposalDateOfShares(),
+        _ => controllers.routes.StartController.start()
```

Comment:
single mixed use should come from the disposal date question in `MixedUsePropertyDetailsController.scala`

Reply:
As I wan unable to find any scenario for SingleMixedUse in CPDP-2588 story, I used this one 
`_ => controllers.routes.StartController.start()`

And also `MixedUsePropertyDetailsController` does not have any disposal date question. 
Please suggest me on this.

Reply:
oh yeah, sorry - not sure what I was looking at. Looks like for single mixed use it uses the same disposal date question in the triage section, so it should go back to there in the last case here